lib/geo_record.rb

```diff
@@ -1,4 +1,19 @@
+require "delegate"
+
 module GeoRecord
+  # Ensure that when coordinates are printed that they are always in decimal degrees,
+  # and not e.g. 4.0e-05
+  # Unfortunately you can't extend Numeric classes directly (e.g. `Coord < Float`).
+  class Coord < DelegateClass(Float)
```

Comment:
Do you mean something like `Coord < Float`? Ruby (or MRI, specifically) doesn't let you do that unfortunately, see e.g. http://www.virtuouscode.com/2011/08/17/you-cant-subclass-integers-in-ruby/ or http://flylib.com/books/en/2.44.1.41/1/  . In short, some built-in classes are special-cased and don't behave like normal ruby classes. The delegate library is there to provide an alternative approach.